config/config.prod.yaml

```diff
@@ -696,12 +696,4 @@
   parameters:
     jira_project_key: FXVPN2
     labels_brackets: both
-    status_map:
-      ASSIGNED: In Progress
-      FIXED: Done
-      WONTFIX: Cancelled
-      DUPLICATE: Cancelled
-      INVALID: Cancelled
-      INCOMPLETE: Done
-      WORKSFORME: Cancelled
-      REOPENED: In Progress
+    status_map: *basic-status-map
```

Comment:
The `*basic-status-map` is defined as https://github.com/mozilla/jira-bugzilla-integration/blob/10a7351e630ab54c63fe418f86ffd6f8fe6bfb36/config/config.prod.yaml#L169-L183